test/functional/core/job_spec.lua

```diff
@@ -78,8 +80,17 @@ describe('jobs', function()
     else
       nvim('command', "let j = jobstart('pwd', g:job_opts)")
     end
+    if iswin() then
+    eq_any({'notification', 'stdout', {0, {dir}}},
+           {'notification', 'stdout', {0, {dir, ''}}},
+           next_msg())
+    eq_any({'notification', 'stdout', {0, {}}},
```

Comment:
This should never happen (empty list)

Reply:
Why does`{dir}` happen? Do we actually expect a empty line to be sent by libuv? Are we not receiving EOF?

Reply:
"lines" doesn't exist as a fundamental concept, from libuv we receive bytes and only bytes. And we unambiguously represent EOF as `{''}`, empty list never happens.

Reply:
So if `{dir}` is received we should expect the next message to be `{''}` (EOF) , right?

Reply:
Not necessarily. If the issue rather is buffering, then in that case we would expect `{'', ''}` and then `{''}`. I think I wrote a helper function exactly for this situation in another test.

Reply:
> I think I wrote a helper function exactly for this situation in another test.

expect_twostreams ?

Reply:
No, expect_twolines, but  I saw it was to limited, it can't handle a final `\n`, only `\n` between two chunks.  Ideally we should join both functions to a generic `expect_streams` that supports _any_ division into chunks as they semantically are equal.